priv/perf/apps/load_test/lib/utils/encoding.ex

```diff
@@ -47,4 +47,27 @@ defmodule LoadTest.Utils.Encoding do
     data = ABI.encode(function_signature, data)
     "0x" <> Base.encode16(data, case: :lower)
   end
+
+  @doc """
+  Produces a stand-alone, 65 bytes long, signature for message hash.
+  """
+  @spec signature_digest(<<_::256>>, <<_::256>>) :: <<_::520>>
+  def signature_digest(hash_digest, private_key_hash) do
```

Comment:
isn't this defied in `LoadTest.Ethereum.Hash` already?

Reply:
Ah yes, so it is 👍